JavaCryptographicArchitecture/src/Cipher.crysl

```diff
@@ -27,6 +29,9 @@ OBJECTS
 	java.nio.ByteBuffer plainBuffer;
 	java.nio.ByteBuffer cipherBuffer;
 	
+	byte[] aad;
+	java.nio.ByteBuffer aadBuffer;
```

Comment:
Same as before, I would suggest a better naming here as well, like `aad_bytebuffer`.

Reply:
I changed it to your suggestion and furthermore changed it for the plaintext and ciphertext bytebuffer aswell to have it consistent.